src/engraving/dom/slur.cpp

```diff
@@ -120,9 +122,15 @@ bool SlurSegment::edit(EditData& ed)
 
     if (ed.key == Key_Home && !ed.modifiers) {
         if (ed.hasCurrentGrip()) {
-            ups(ed.curGrip).off = PointF();
+            startEditDrag(ed);
```

Comment:
ensures operation is undoable

Reply:
I see that in the master the undo/redo works without this code

Reply:
Not when I tested it!  To be clear, it requires you adjust a handle position, ensure handle is selected and press Home, then undo, which should cause it to return to the adjusted position - NOT the original position.  After all, you could perform another action between adjusting handle and using home to reset it.

Reply:
Okay, it works without this code for normal moves, so It is interesting why it doesn’t work for Home and we need to add this code here.

Reply:
Because other key strokes are handled by NotationInteraction::handleKeyPress( ) where it does use startEditDrag( ) and endEditDrag( ).

Reply:
If we move this code out of edit and move it to editDrag, will everything work as expected?
```
    bool handled = m_editData.element->edit(m_editData); // will return false for Home key
    if (!handled) {
        handled = handleKeyPress(event); // calls startEditDrag/editDrag/endEditDrag
    }
```

Reply:
No, because for the undo & redo to work correctly you must first call startEditDrag(), *then* reset the values, then call endEditDrag().

Reply:
we do it in `handleKeyPress`
(for other contributors to know why this PR is not approved and merged)

Reply:
Ok, well, thanks for letting me know, but I would have been happy to modify my PR to make this change if that's what was agreed by all as being the best way to proceed...

Reply:
So somebody else is implementing this and we should abandon this PR? @Eism